src/sa_web/static/js/views/place-form-view.js

```diff
@@ -111,8 +111,8 @@ var Shareabouts = Shareabouts || {};
           animationDelay = 400;
 
       this.selectedCategory = $(evt.target).parent().prev().attr('id'),
-      this.selectedDatasetId = this.options.placeConfig.place_detail[this.selectedCategory].dataset,
-      this.selectedDatasetSlug = this.options.placeConfig.place_detail[this.selectedCategory].datasetSlug;
+      this.selectedDatasetId = this.options.placeConfig.categories[this.selectedCategory].dataset,
```

Comment:
I think I found a minor bug here. It's a simple fix though - I think we should change `categories` back to `place_detail`, or else we get a javascript error.

This comes up when choosing a category on the place form.